test/parallel/test-child-process-stdout-flush.js

```diff
@@ -1,29 +1,29 @@
 'use strict';
-var common = require('../common');
-var assert = require('assert');
-var path = require('path');
-var spawn = require('child_process').spawn;
-var sub = path.join(common.fixturesDir, 'print-chars.js');
+const common = require('../common');
+const assert = require('assert');
+const path = require('path');
+const spawn = require('child_process').spawn;
+const sub = path.join(common.fixturesDir, 'print-chars.js');
 
-var n = 500000;
+const n = 500000;
 
-var child = spawn(process.argv[0], [sub, n]);
+const child = spawn(process.argv[0], [sub, n]);
 
-var count = 0;
+let count = 0;
 
 child.stderr.setEncoding('utf8');
-child.stderr.on('data', function(data) {
+child.stderr.on('data', (data) => {
   console.log('parent stderr: ' + data);
   assert.ok(false);
 });
 
 child.stdout.setEncoding('utf8');
-child.stdout.on('data', function(data) {
+child.stdout.on('data', (data) => {
   count += data.length;
   console.log(count);
 });
 
-child.on('close', function(data) {
-  assert.equal(n, count);
+child.on('close', common.mustCall((data) => {
+  assert.strictEqual(n, count);
   console.log('okay');
-});
+}));
```

Comment:
The `close`  event listener has two arguments: `code` and `signal`. I would check that `code` is `0` and `signal` is not defined.